content/en/apps/concepts/interoperability.md

```diff
@@ -3,36 +3,52 @@ title: Interoperability
 linkTitle: "Interoperability"
 weight: 8
 description: >
-  Exchanging information between CHT Core and other systems
+  Two way data synchronization between the CHT Core and other health systems. 
```

Comment:
```suggestion
  Exchanging information between the CHT Core and other health systems 
```

I think information exchange is more accurate than synchronization in the context of interoperability.